src/snowcli/cli/nativeapp/commands.py

```diff
@@ -71,3 +75,24 @@ def app_bundle(
     manager = NativeAppManager(project_path)
     manager.build_bundle()
     return MessageResult(f"Bundle generated at {manager.deploy_root}")
+
+
+@app.command("diff", hidden=True)
+@with_output
+@global_options_with_connection
+def nativeapp_stage_diff(
+    stage_fqn: str = typer.Argument(None, help="Name of stage"),
+    folder_name: str = typer.Argument(None, help="Path to local folder"),
+    **options,
+) -> MessageResult:
+    """
+    Diffs a stage with a local folder
+    """
+    diff: DiffResult = stage_diff(Path(folder_name), stage_fqn)
+    output = f"""\
+        only local:  {', '.join(diff.only_local)}
+        only stage:  {', '.join(diff.only_on_stage)}
+        mod/unknown: {', '.join(diff.different)}
+        identical:   {', '.join(diff.identical)}"""
```

Comment:
Have you though about using `ObjectResult`? In this way the output will be easier to programmatically parse, and for human readable we will get a table.

Reply:
Cam's currently OOO, so I'm making the changes to ack the feedback. 
I have updated with ObjectResult, and IIUC, Table is the default output format so it should print as a table.